woocommerce-gutenberg-products-block.php

```diff
@@ -375,19 +375,20 @@ function wgpb_print_script_settings() {
 	// Will need `orderStatuses` if the OrderStatus component is used.
 	// Deliberately excluding: `embedBreadcrumbs`, `trackingEnabled`.
 	$settings = array(
-		'adminUrl'      => admin_url(),
-		'wcAssetUrl'    => plugins_url( 'assets/', WC_PLUGIN_FILE ),
-		'siteLocale'    => esc_attr( get_bloginfo( 'language' ) ),
-		'currency'      => array(
+		'adminUrl'          => admin_url(),
+		'wcAssetUrl'        => plugins_url( 'assets/', WC_PLUGIN_FILE ),
+		'siteLocale'        => esc_attr( get_bloginfo( 'language' ) ),
+		'currency'          => array(
 			'code'      => $code,
 			'precision' => wc_get_price_decimals(),
 			'symbol'    => get_woocommerce_currency_symbol( $code ),
 			'position'  => get_option( 'woocommerce_currency_pos' ),
 		),
-		'stockStatuses' => wc_get_product_stock_status_options(),
-		'siteTitle'     => get_bloginfo( 'name' ),
-		'dataEndpoints' => array(),
-		'l10n'          => array(
+		'stockStatuses'     => wc_get_product_stock_status_options(),
+		'siteTitle'         => get_bloginfo( 'name' ),
+		'placeholderImgSrc' => wc_placeholder_img_src(),
```

Comment:
This should be added to the `$block_settings` array just below this, which is used for the `wc_product_block_data` global. We don't use `wcSettings` directly in this project, it's just here to not cause errors with using `@woocommerce/components`